src/main/java/fish/payara/jcache/api/NamedCache.java

```diff
@@ -55,62 +55,63 @@
 public @interface NamedCache {
```

Comment:
I would also include [PayaraGeneratedCacheKey](http://payara.github.io/Payara/appserver_API/New%20folder/payara-appserver-modules_api/payara-jsr107_api/site/apidocs/fish/payara/cdi/jsr107/impl/PayaraGeneratedCacheKey.html) in the API, as it is necessary to combine programmatic and annotation-based caches.